astrocut/cube_cut.py

```diff
@@ -815,16 +815,22 @@ def cube_cut(self, cube_file, coordinates, cutout_size,
 
         warnings.filterwarnings("ignore", category=wcs.FITSFixedWarning)
         fits_options: Dict[str, Any] = {"mode": "denywrite", "lazy_load_hdus": True}
+
         if cube_file.startswith("s3://"):
             fits_options["use_fsspec"] = True
-            # block size should be:
-            # block_size <= m * hdul[1].section.shape[2] * hdul[1].section.shape[3] * 4 bytes
             fits_options["fsspec_kwargs"] = {"default_block_size": 10_000, "anon": True}
             # only use .section for remote data
             cube_data_prop = "section"
+
+            # block size should be:
+            # block_size <= m * hdul[1].section.shape[2] * hdul[1].section.shape[3] * 4 bytes
+            with fits.open(cube_file, **fits_options) as cube:
+                block_size = cube[1].section.shape[1] * cube[1].section.shape[2] * cube[1].section.shape[3] * 4
```

Comment:
just confirming this should always be `1` here? it doesn't matter if it's TICA or SPOC?

```python
                block_size = cube[1].section.shape[1] * cube[1].section.shape[2] * cube[1].section.shape[3] * 4
```

Reply:
Correct, I designed `TicaCubeFactory` so that the TICA cubes are structurally identical to the SPOC cubes. EXT1 is always the SCI extension.